#! /bin/sh /usr/share/dpatch/dpatch-run
# 51_texinfo_info_typo_fix.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix a typo in the texinfo.info file

@DPATCH@
diff -ur texinfo/doc/texinfo.txi.orig texinfo/doc/texinfo.txi
--- doc/texinfo.txi.orig	2005-10-20 16:02:10.000000000 +0200
+++ doc/texinfo.txi	2005-10-20 16:02:18.000000000 +0200
@@ -1355,7 +1355,7 @@
 @cindex Required in Texinfo file
 @cindex Texinfo file minimum
 
-By convention, the namea of a Texinfo file ends with (in order of
+By convention, the name of a Texinfo file ends with (in order of
 preference) one of the extensions @file{.texinfo}, @file{.texi},
 @file{.txi}, or @file{.tex}.  The longer extensions are preferred since
 they describe more clearly to a human reader the nature of the file.
